content/docs/tutorials/get-started/data-pipelines.md

```diff
@@ -0,0 +1,335 @@
+# Connect Code and Data
+
+Even in its basic scenarios, commands like `dvc add`, `dvc push`, `dvc pull`
+described in the previous page could be used independently and provide a basic
+useful framework to track, save and share models and large data files. To
+achieve full reproducibility though, we'll have to connect code and
+configuration with the data it processes to produce the result.
+
+<details>
+
+### Expand to prepare example code
+
+If you've followed this _Get Started_ tutorial from the beginning, run these
+commands to get the example code:
+
+```dvc
+$ wget https://code.dvc.org/get-started/code.zip
+$ unzip code.zip
+$ rm -f code.zip
+```
+
+Windows doesn't include the `wget` utility by default, but you can use the
+browser to download `code.zip`. (Right-click
+[this link](https://code.dvc.org/get-started/code.zip) and select
+`Save Link As...` (Chrome). Save it into the project directory.
```

Comment:
I wonder if this note is helpful. Windows also doesn't have `unzip` built-in...

> This same happens in other 2 unrelated docs where this note is repeated for `wget` but they also use `unzip` or `du`.

Reply:
Nor does it support \ new line continuations like used in commands further down.

Reply:
@shcheklein per https://github.com/iterative/example-repos-dev/issues/13#issuecomment-614320329 should we remove these notes even if we replace zip for tar?

Reply:
Well, maybe not from the Get Started, but the note is in a couple other places (more advanced guides)

Reply:
let you decide on this :) can with the simplest option and focus on *nix ... will be improving after that

Reply:
Decided to keep this as a note here but removed from somewhere else. See 1a4ebcba